scripts/test/Muon/duel_plot_maxent_pane/duel_plot_maxent_pane_model_test.py

```diff
@@ -0,0 +1,324 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+from unittest import mock
+from Muon.GUI.Common.ADSHandler.workspace_naming import (MAXENT_STR, get_maxent_workspace_name, get_raw_data_workspace_name,
+                                                         RECONSTRUCTED_SPECTRA)
+from Muon.GUI.Common.test_helpers.context_setup import setup_context
+from Muon.GUI.Common.utilities.algorithm_utils import create_empty_table
+from Muon.GUI.Common.muon_group import MuonGroup
+from Muon.GUI.Common.plot_widget.raw_pane.raw_pane_model import RawPaneModel
+from Muon.GUI.Common.plot_widget.data_pane.plot_data_pane_model import PlotDataPaneModel
+from Muon.GUI.FrequencyDomainAnalysis.plot_widget.duel_plot_maxent_pane.duel_plot_maxent_pane_model import DuelPlotMaxentPaneModel
+from mantidqt.utils.qt.testing import start_qapplication
+
+
+def gen_table():
+    tab = create_empty_table("table")
+    tab.addColumn('str', 'Group')
+    tab.addColumn('str', 'Detectors')
+    groups = ["fwd", "bwd"]
+    detectors_list = ["0,1,2,3,4", "7,8,9,10"]
+    for j, name in enumerate(groups):
+        detectors = detectors_list[j]
+        tab.addRow([name, detectors])
+    return tab
+
+
+@start_qapplication
```

Comment:
I've just noticed this, we shouldn't be trying to start Qt applications in our model tests
Can we remove it? As it's usually a pre-cursor to a flaky test